### DISCUSSION TITLE
p/48334b653bc9470aae86d06519a907ec/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#8] [Announcements]
# p/48334b653bc9470aae86d06519a907ec/

2021 年终总结及 2022 的计划  年终总结具体来讲  补足和实践理论层面的知识 – 看完了 sicp 课程 继续维护和改进 liuli-tools/liuli-utils – 继续维护中 继续维护 joplin-utils 系列的笔记相关工具 – joplin-vscode-plugin 发布 1.0，从 joplin-blog 衍生出 mami 提高发布博客的质量，而不是简单的笔记映射

https://blog.rxliuli.com/p/48334b653bc9470aae86d06519a907ec/

**Comment by ryan4yin** [comment]
总结写得好早啊，11 月底就写了

**Reply by rxliuli** [reply]
嗯，动笔写的时间比较早，不过发布也是上周的事情

**Reply by ryan4yin** [reply]
我也在考虑我的年终总结，不过到元旦前，我觉得自己还能再冲刺一波自己的「年度绩效」 emmm

**Reply by rxliuli** [reply]
23333
